admin/src/js/controllers/edit-user.js

```diff
@@ -254,24 +277,57 @@ angular
       return hasPlace && hasContact;
     };
 
+    const validateFacilityHierarchy = () => {
+      const placeIds = $scope.editUserModel.place;
+      const docsFetched = 10;
```

Comment:
It's unclear why we are fetching less docs than we are actually selecting?

Reply:
Right, I forgot to take it out after some experimentation.